eQuiz/modules/eQuiz.Web/Areas/Admin/Scripts/equiz.layoutController.js

```diff
@@ -1,27 +1,17 @@
 ﻿(function (angular) {
```

Comment:
Rename the file according to https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#naming